docs/get-started/python.md

```diff
@@ -0,0 +1,127 @@
+   * The `synth` method is called on the `app` instance, which produces the required Kubernetes YAML manifest files based on the defined resources. Note that in this example, we haven't defined any resources within the `MyChart` constructor, so running the "cdk8s synth" command in the CLI would generate a blank Kubernetes manifest.
+
+## Defining Kubernetes resources
+Now, we are ready to define Kubernetes resources for our application. In this sample, we illustrate the creation of a basic Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) for a sample application. We start by importing the imports directory and the **k8s** sub-directory, which contains an `__init__.py` file where all CDK8s Kubernetes classes and functions are defined.
+
+### Copy the code sample
+1. Copy and paste the following code sample into the existing `main.py` file of your project.
+```python
+#!/usr/bin/env python
+from cdk8s import App, Chart
+from constructs import Construct
+from imports import k8s
+
+
+class AwesomeApp(Chart):
+   def __init__(self, scope: Construct, ns: str, app_label: str):
+       super().__init__(scope, ns)
+
+       # Define a Kubernetes Service
+       k8s.KubeService(self, "my-service",
+                       spec=k8s.ServiceSpec(
+                           selector={"app": app_label},
+                           ports=[k8s.ServicePort(
+                               port=80, target_port=k8s.IntOrString.from_number(8080))],
+                           type="LoadBalancer"
+                       )
+                    )
+
+
+app = App()
+AwesomeApp(app, "getting-started", app_label="my-app")
+
+app.synth()
+```
+
+A few things worth noting about this sample:
+
+- An `AwesomeApp` class is created by passing the app instance, a string identifier "getting-started", and a custom `app_label` ("my-app") as arguments. This action generates and registers the Kubernetes resources defined in the `AwesomeApp` class within the application. We use the custom `app_label` parameter to dynamically set the "app" key in the "label" selectors and "metadata" labels for our Kubernetes resources.
+- The `__init__` method of the `AwesomeApp` class creates a Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) by instantiating the `k8s.KubeService` class with the following parameters: `self` as the scope of the Service resource within the `AwesomeApp` class, a string identifier "my-service", and a `k8s.ServiceSpec` object that defines the Service specification with a selector dictionary having the key "app" and the value set to the `app_label` parameter, a list of ports containing a single `k8s.ServicePort` object with `port` set to 80 and `target_port` set to 8080, and a `type` parameter with the value "LoadBalancer". The `app_label` parameter is passed to the `AwesomeApp` class.
```

Comment:
This is hard to follow. Its basically repeating exactly what the code does and I don't think adds much value. We expect people to know python, and know a bit of Kubernetes...I think we can just drop this.

Reply:
See above comment.